src/opendr/perception/multimodal_human_centric/audiovisual_emotion_learner/algorithm/efficientface_modulator.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This code is modified from: https://github.com/zengqunzhao/EfficientFace/blob/master/models/modulator.py

```

Comment:
@ad-daniel Should we also list here the original license of the file (MIT)?

Reply:
Should I replace the OpenDR license or add it?

Reply:
I think in this case we add both of them, but we can make sure in the TMT meeting.

Reply:
Yes, the original copyright should be maintained. If the modifications done are very minor, then it's fine to leave only that license (and add this file/folder to the list of skipped directories in `test_license.py`).
If you deem the changes to be important enough to warrant a copyright notice, then you should add also the OpenDR license (i.e. both should be mentioned)